tools/manifest/download.py

```diff
@@ -33,46 +33,53 @@ def should_download(manifest_path, rebuild_time=timedelta(days=5)):
     return False
 
 
-def git_commits(repo_root):
+def merge_pr_tags(repo_root, max_count=50):
```

Comment:
This can't be relanded as-is even if it wasn't the cause of the failures, because tags aren't fetched in the Taskcluster setup: https://github.com/web-platform-tests/wpt/issues/14207#issuecomment-441189804